build/scripts/build.sh

```diff
@@ -68,8 +77,11 @@ echo "TNS_IPA_SIZE: "$GAMERAWW_IPA_SIZE"KB"
 echo "TNS_IPA_SIZE_KB\\n"$GAMERAWW_IPA_SIZE > "$WORKSPACE/build-stats.csv"
 
 echo "Building TestRunner..."
-xcodebuild -configuration Debug -sdk iphoneos -target TestRunner ARCHS="armv7" ONLY_ACTIVE_ARCH=NO  >> "$WORKSPACE/build.log" 2>&1
+xcodebuild_pretty -configuration Debug -sdk iphoneos -target TestRunner ARCHS="armv7" ONLY_ACTIVE_ARCH=NO
 echo "Packaging TestRunner..."
 xcrun -sdk iphoneos PackageApplication -v "$WORKSPACE/cmake-build/tests/TestRunner/Debug-iphoneos/TestRunner.app" \
     -o "$WORKSPACE/cmake-build/tests/TestRunner/Debug-iphoneos/TestRunner.ipa" \
-         >> "$WORKSPACE/build.log" 2>&1
\ No newline at end of file
+         >> "$WORKSPACE/build.log" 2>&1
+TestRunner/Debug-iphoneos/TestRunner.app" \
```

Comment:
What's the purpose of the following three lines? It seems to fail the build locally.